app/src/main/res/values/strings.xml

```diff
@@ -79,8 +79,8 @@
   <string name="about_license">The Wikimedia Commons app is an open-source app created and maintained by grantees and volunteers of the Wikimedia community. The Wikimedia Foundation is not involved in the creation, development, or maintenance of the app. </string>
   <string name="trademarked_name" translatable="false">Wikimedia Commons</string>
   <string name="about_improve">Create a new &lt;a href=\"https://github.com/commons-app/apps-android-commons/issues\"&gt;GitHub issue&lt;/a&gt; for bug reports and suggestions.</string>
-  <string name="about_privacy_policy"><u>Privacy policy</u></string>
-  <string name="about_credits"><u>Credits</u></string>
+  <string name="about_privacy_policy"><![CDATA[<u>Privacy policy</u>]]></string>
+  <string name="about_credits">[<![CDATA[<u>Credits</u>]]></string>
```

Comment:
Is the first `[` really needed?

Reply:
@nicolas-raoul  Yup, the first [ is part of the syntax.
Source: https://en.wikipedia.org/wiki/CDATA